.travis.yml

```diff
@@ -172,16 +176,30 @@ jobs:
     after_success:
       - echo "Xilinx Vivado metapackages built for versions:" && cat $VERSIONS
 
+ # Move packages from the current label to main
```

Comment:
I don't see this running on https://travis-ci.com/github/litex-hub/litex-conda-eda/builds/175201099 ?

Reply:
only on master